src/contracts/core/DelegationManager.sol

```diff
@@ -293,23 +283,21 @@ contract DelegationManager is
         uint256 existingDepositShares,
         uint256 addedShares
     ) external onlyStrategyManagerOrEigenPodManager {
-        // if the staker is delegated to an operator
-        if (isDelegated(staker)) {
-            address operator = delegatedTo[staker];
-            IStrategy[] memory strategies = new IStrategy[](1);
-            strategies[0] = strategy;
-            uint64[] memory maxMagnitudes = allocationManager.getMaxMagnitudes(operator, strategies);
+        address operator = delegatedTo[staker];
 
-            // add deposit shares to operator's stake shares and update the staker's depositScalingFactor
-            _increaseDelegation({
-                operator: operator,
-                staker: staker,
-                strategy: strategy,
-                existingDepositShares: existingDepositShares,
-                addedShares: addedShares,
-                maxMagnitude: maxMagnitudes[0]
-            });
-        }
+        IStrategy[] memory strategies = new IStrategy[](1);
+        strategies[0] = strategy;
+        uint64 maxMagnitude = allocationManager.getMaxMagnitudes(operator, strategies)[0];
+
+        // Increase the staker's deposit scaling factor and delegate shares to the operator
+        _increaseDelegation({
+            operator: delegatedTo[staker],
```

Comment:
reuse `operator` instead of reading storage again

Reply:
[34ba24b](https://github.com/Layr-Labs/eigenlayer-contracts/pull/876/commits/34ba24be2ffa25c33f8c45803a56bb9377010a31)